package.json

```
@@ -84,7 +91,8 @@
     "test:log": "jest --silent=false",
     "lint": "standard",
     "lint:write": "standard --fix",
-    "lint:ts": "tsc"
+    "lint:ts": "tsc",
+    "lint:styles": "stylelint \"src/{assets/scss,app}/**/*.{css,scss}\" --quiet-deprecation-warnings"
```

Is there a reason that we are ignoring `src/assets/crubrand/`, `src/assets/cru-scss/`, and `src/common/components/loading/loading.scss`? If that's intentional, can you run the lint on everything and put the paths to ignore in a `.stylelintignore`? That way, devs can install the Stylelint VS Code extension and have it auto-fix problems on save, while ignoring any SCSS files that we don't want to apply our style rules to.

```suggestion
    "lint:styles": "stylelint 'src/**/*.scss' --quiet-deprecation-warnings"
```

---

It was intentional to leave out `src/assets/crubrand/`, `src/assets/cru-scss/` as it took a lot to do `src/assets/scss`.
When running it on `'src/**/*.scss'`, it updates 161 files and still returns over 3000 errors that I would have to manually edit.

I think that is best for another PR.

---

That's totally fine. I'm just asking that we put ignored paths in a `.stylelintignore` file so that the Stylelint VS Code extension doesn't try to auto-fix them on save.

---

I've added an ignore property to `stylelint.config.mjs` so I can run styleLint on `src/**/*.{css,scss}` and capture all SCSS/CSS files which includes `src/common/components/loading/loading.scss`.

---

Is there a reason you don't want to add a `.stylelintignore` file and put the ignore paths there instead? Try editing the `crubrand` SCSS file with the Stylelint VS Code plugin installed and the configuration from #1162 and you can see that it will auto-format everything, which we don't want. Maybe I'm wrong, but it seems like devs would want to use that extension to auto-format the SCSS files that we have formatting turned on for.

---

So when I use the `.vscode/settings`, it ignores the folders I've put in the ignore property in `stylelint.config.mjs`.

---

Other files do get auto-formated, but not `src/assets/crubrand/` and `src/assets/cru-scss/`

---

You're right. I apologize, I must have missed something during testing or neglected to restart Stylelint after changing the config file.